### PR TITLE
Adding new supported states to Ios_lldp_interfaces resource module

### PR DESCRIPTION
**Pull request by justjais**
##### SUMMARY

Adding new supported states to Ios_lldp_interfaces resource module

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

Ios_lldp_interfaces

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

File: plugins/modules/ios_lldp_interfaces.py [hunk 1]
```diff
@@ -429,7 +440,143 @@
...
+#    Tx state: IDLE
+#    Rx state: WAIT FOR FRAME
+#
+# GigabitEthernet0/2:
+#    Tx: enabled
+#    Rx: enabled
+#    Tx state: IDLE
+#    Rx state: WAIT FOR FRAME
+
+- name: Gather listed LACP interfaces with provided configurations
```
Comment (rohitthakur2590) [inline plugins/modules/ios_lldp_interfaces.py:468]: replace LACP with LLDP

File: plugins/modules/ios_lldp_interfaces.py [hunk 2]
```diff
@@ -452,6 +599,7 @@
   type: list
   sample: ['interface GigabitEthernet 0/1', 'lldp transmit', 'lldp receive']
 """
+
```
Comment (rohitthakur2590) [inline plugins/modules/ios_lldp_interfaces.py:602]: required_if = [
        ("state", "merged", ("config",)),
        ("state", "replaced", ("config",)),
        ("state", "rendered", ("config",)),
        ("state", "overridden", ("config",)),
        ("state", "parsed", ("running_config",)),
    ]

    mutually_exclusive = [("config", "running_config")]

File: plugins/modules/ios_lldp_interfaces.py [hunk 3]
```diff
@@ -429,7 +440,143 @@
...
+  cisco.ios.ios_lldp_interfaces:
+    running_config:
+      "interface GigabitEthernet0/1
+       lacp port-priority 10
+       interface GigabitEthernet0/2
+       lacp port-priority 20
+       interface Port-channel10
+       lacp max-bundle 2
+       slacp fast-switchover"
+    state: parsed
```
Comment (rohitthakur2590) [inline plugins/modules/ios_lldp_interfaces.py:557]: I think this got missed by me.

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Comment by justjais** [comment]
recheck

**Pull request merged**